src/App.js

```diff
@@ -27,8 +28,8 @@ function App() {
     <BrowserRouter>
       <Routes>
         <Route path="/" element={<Layout />}>
-          <Route index element={<div>This will be the home</div>} />
-          <Route path="categories" element={<div>Not implemented yet</div>>} />
+          <Route index element={<div><BooksLogic /></div>} />
+          <Route path="categories" element={<div>Gabriel Rozas</div>} />
```

Comment:
Nice job so far👍. You are required in this milestone to have two page-components😉,  One that renders all the actions concerning the books, which you already created, and another for the `categories` which you haven't created😉.

Kindly add a component for categories which should be a page😉. Add the routes in your nav. so on your nav bar you can move to the categories page too😉.

<img width="703" alt="image" src="https://github.com/EmiLoKiJu/bookstore/assets/64643995/9c4b19b6-fc87-46a2-8e44-dc7119d91cc2">

In doing this, kindly create a`Nav` component for your `navbar` which will carry the links to the home page (books page) and the categories page😉.